R/PictureBlock.R

```diff
@@ -40,7 +40,7 @@ PictureBlock <- R6::R6Class( # nolint: object_name_linter.
           if (inherits(content, "grob")) {
             grid::grid.newpage()
             grid::grid.draw(content)
-          } else if (inherits(content, "gg")) {
+          } else if (inherits(content, "gg") || inherits(content, "Heatmap")) {
```

Comment:
unlike `is`, `inherits` accept a vector 
```suggestion
          } else if (inherits(content, c("gg", "Heatmap")) {
```